docs/reference/how-to/avoid-oversharding.asciidoc

```diff
@@ -0,0 +1,73 @@
+
+[float]
+=== How to reduce shard counts and increase shard size
+
+Try these six methods to reduce oversharding.
+
+[float]
+==== 1. Reduce the number of shards for new indices
+
+The `index.number_of_shards` setting can be specified for new indices created with the <<indices-create-index,Create Index API>> or as part of <<indices-templates,index templates>> for indices automatically created by <<index-lifecycle-management,Index Lifecycle Management>>. For Elasticsearch versions below 7.0, the default value for this setting was `5`. For versions 7.0 and higher, the default value for this setting is `1`.
+
+The <<rollover-index-api-example,rollover request>> body can also override that same `index.number_of_shards` setting.
+
+[float]
+==== 2. Create larger shards by increasing the rollover thresholds
+
+If indices are rolled over by either directly using the Rollover API or by specifying the Index Lifecycle Management {ref}/_actions.html#ilm-rollover-action[Rollover action], then the rollover condition thresholds (`max_age`, `max_docs`, `max_size`) can be increased to allow the indices to grow to a larger size before being rolled over, thereby generating larger shards.
+
+Take special note of any indices that are empty. These may be managed by an ILM policy that is rolling over the indices because the `max_age` threshold is met. In this case, you may need to adjust the policy to make use of the `max_docs` or `max_size` properties to prevent the creation of these empty indices. One example where this may happen is if one or more Beats stop sending data. If the ILM-managed indices for those Beats are configured to rollover daily, then new, empty indices will be generated each day. Empty indices can be identified using the following {ref}/cat-count.html[cat count API]: `GET /_cat/count?v`.
+
+
+[float]
+==== 3. Create larger shards by using index patterns spanning longer time periods
+
+Creating indices covering longer time periods will also reduce the index, and thereby shard, counts and increase index sizes. For example, instead of daily indices, create monthly, or even yearly indices.
```

Comment:
```suggestion
Creating indices covering longer time periods reduces index and shard counts while increasing index sizes. For example, instead of daily indices, you can create monthly, or even yearly indices.
```